src/config/settings.ts

```diff
@@ -429,7 +429,8 @@ const settings: Settings = {
     pools: {
       [ChainsIds.ETH]: {
         [DexProtocolName.UNISWAP_V3]: {
-          helperContractId: <string>process.env['UNISWAPV3_HELPER_CONTRACT_ID'],
+          helperContractId: <string>process.env.UNISWAPV3_HELPER_CONTRACT_ID,
+          updateLimitTimeInMs: parseInt(process.env.UNISWAPV3_HELPER_CONTRACT_ID || String(getDayInMillisecond(1))),
```

Comment:
`updateLimitTimeInMs` what is update time?

Reply:
Removed, this is for next PR